examples/website/i3s-colorization-by-attributes/src/types.ts

```diff
@@ -0,0 +1,16 @@
+export type COLOR = [number, number, number, number];
```

Comment:
CAPS is for constants

```suggestion
export type Color = [number, number, number, number];
```

Reply:
Removed type definition